albatross-1/README.md

```diff
@@ -0,0 +1,207 @@
+# Albatross Testnet Instructions
+
+## Software Requirements
+
+- Go version v1.14.+
+- Stakebird version : [v0.3.0](https://github.com/public-awesome/stakebird/releases/tag/v0.3.0)
+
```

Comment:
Technically, `make` and a gcc compiler are also dependencies... though most people will already have these unless they are starting with a fresh server instance.

Reply:
Yeah started a separate doc on setting up an Ubuntu node from scratch: https://www.notion.so/publicawesome/Setting-up-a-Stakebird-Validator-on-Ubuntu-56ab9f4eb1b14fd6a7dc63ee0d763093. Will include that here or post it to the blog.

Reply:
Actually might just link to the binary, since we're building for all platforms anyway.